templates/news/news-en.html

```diff
@@ -3,224 +3,476 @@
 			</figcaption>
 		</figure>
 	</a>
-	<a href="#" class="hide">
+	<a href="/en/services/health/campaigns/childhood-vaccines.html" class="promo hide learmore" data-gc-analytics="promo:homePageFeature">
 		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/7834BC/FFFFFF.png" alt="">
+			<img class="img-responsive full-width" src="https://dummyimage.com/1170x347/7834BC/FFFFFF.png" alt="">
 			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Protect your child with routine vaccination</span>
 			</figcaption>
 		</figure>
 	</a>
-	<a href="#" class="hide">
+	<a href="https://natural-resources.canada.ca/simply-science/forestry-and-insects/667?utm_campaign=ss-nfw&amp;utm_medium=vurl&amp;utm_source=canada.ca&amp;utm_content=forestry-en" class="promo hide learmore" data-gc-analytics="promo:homePageFeature">
 		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/0000FF/000000.png" alt="">
+			<img class="img-responsive full-width" src="https://dummyimage.com/1170x347/0000FF/000000.png" alt="">
 			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Discover groundbreaking forestry research at Natural Resources Canada</span>
 			</figcaption>
 		</figure>
 	</a>
 </div>
+<!-- END -->
+
 <div class="clearfix">
 	<div class="pull-right">
 		{% include gc-follow-us/follow.html %}
 	</div>
 </div>
-{% include components/gc-features/features.html %}
+
+<section class="gc-features">
+	<h2>Features</h2>
+	<!-- START: Add the list of links to the features to display in basic HTML mode -->
+	<ul class="nojs-show">
+		<li><a href="#">[1. Feature hyperlink text]</a></li>
+		<li><a href="#">[2. Feature hyperlink text]</a></li>
+		<li><a href="#">[3. Feature hyperlink text]</a></li>
+		<li><a href="#">[4. Feature hyperlink text]</a></li>
+		<li><a href="#">[5. Feature hyperlink text]</a></li>
+		<li><a href="#">[6. Feature hyperlink text]</a></li>
+	</ul>
+	<!-- END -->
+
+	<!-- Add the "data-wb-randomize" attribute to the "row" element to randomize the features -->
+	<div class="row wb-eqht-grd">
+		<div class="col-lg-4 col-sm-6 hght-inhrt" data-wb-randomize='{ "toggle": "hide" }'>
+			<div class="well well-sm eqht-trgt hide">
+				<img src="../assets/news-feature-1-360x203.jpg" alt="">
+				<h3><a href="#" class="stretched-link">[1. Feature hyperlink text]</a></h3>
+				<p>1. Brief description of the feature being promoted.</p>
+			</div>
+			<div class="well well-sm eqht-trgt hide">
+				<img src="../assets/news-feature-2-360x203.jpg" alt="">
+				<h3><a href="#" class="stretched-link">[2. Feature hyperlink text]</a></h3>
+				<p>2. Brief description of the feature being promoted.</p>
+			</div>
```

Comment:
```suggestion
			<section class="well well-sm eqht-trgt hide">
				<img src="../assets/news-feature-1-360x203.jpg" alt="">
				<h3><a href="#" class="stretched-link">[1. Feature hyperlink text]</a></h3>
				<p>1. Brief description of the feature being promoted.</p>
			</section>
			<section class="well well-sm eqht-trgt hide">
				<img src="../assets/news-feature-2-360x203.jpg" alt="">
				<h3><a href="#" class="stretched-link">[2. Feature hyperlink text]</a></h3>
				<p>2. Brief description of the feature being promoted.</p>
			</section>
```
See my rational in my previous comment